content/4_Gold/SP.mdx

```diff
@@ -358,13 +358,11 @@ public class test {
 
 <Warning>
 
-It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem!
+It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem! 
 
-<IncompleteSection>
+The last testcase contains 100000 destinations and 149997 routes. City 1 has around 50000 flights to cities 2 through 50000. Cities 2 through 50000 have flights that go to city 50001. City 50001 has flights that go to cities 50002 through 100000. Without the `continue`, after the program pops off cities 1 through 50000, the priority queue will contain 49999 routes that point to city 50001. After getting the shortest route to city 50001 from these routes, it will move on to the cities 50002 through 100000 that have flights from city 50001. This means that city 50001 will be processed 49998 more times than needed; furthermore, it carries over to the 49999 other flights from city 50001 to cities 50002-100000. This results in a worse runtime than $\Theta(N^2\log N)$, which will TLE. 
```

Comment:
"149997 routes" ==> "149997 flights" (?)

Reply:
yes, ill fix that